frontend/src/lib/components/LemonCalendar/LemonCalendarRange.tsx

```diff
@@ -69,18 +85,20 @@ export function LemonCalendarRange({ value, onChange, onClose, months }: LemonCa
 
     return (
         <div className="LemonCalendarRange" data-attr="lemon-calendar-range">
-            <div className="flex justify-between border-b p-2 pb-4">
-                <h3 className="mb-0">Select a fixed time period</h3>
-                {onClose && (
-                    <LemonButton
-                        icon={<IconClose />}
-                        size="small"
-                        status="stealth"
-                        onClick={onClose}
-                        aria-label="close"
-                    />
-                )}
-            </div>
+            {hideHeader ? null : (
+                <div className="flex justify-between border-b p-2 pb-4">
+                    <h3 className="mb-0">Select a fixed time period</h3>
+                    {onClose && (
+                        <LemonButton
+                            icon={<IconClose />}
+                            size="small"
+                            status="stealth"
+                            onClick={onClose}
+                            aria-label="close"
+                        />
+                    )}
+                </div>
+            )}
             <div className="p-2">
```

Comment:
Without a header/footer, this `p-2` added just enough padding to break the flow, and shouldn't be there.

![2022-09-01 16 50 10](https://user-images.githubusercontent.com/53387/188130117-a4330052-eed1-4429-b0de-1b513e30d02a.gif)